README.md

```diff
@@ -24,25 +34,27 @@ For Slack platform, you need to provide the following required variables:
 * **SLACK_APP_TOKEN**. Slack App-Level Token
 * **SLACK_BOT_TOKEN**. Slack Bot User OAuth Token
 
-    > If you want to use the AccessBot resource access form, please read this [docs](docs/slack/CONFIGURE_ACCESSBOT_FORM.md). 
-
 ### MS Teams Installation
 For MS Teams platform, you need to provide the following required variables:
 * **AZURE_APP_ID**. Azure Bot application ID
 * **AZURE_APP_PASSWORD**. Azure Bot application password
 
 For a full list of configuration variables please read: [Configure AccessBot](docs/configure_accessbot/CONFIGURE_ACCESSBOT.md)
 
-Detailed instructions about how to configure Slack and SDM for AccessBot can be found here:
-* [Configure Slack](docs/slack/CONFIGURE_SLACK.md)
+Detailed instructions about how to configure SDM and a platform (Slack, Slack Classic or MS Teams) for AccessBot can be found here:
 * [Configure SDM](docs/configure_accessbot/CONFIGURE_SDM.md)
+* [Configure Slack](docs/slack/CONFIGURE_SLACK.md)
+* [Configure Slack Classic](docs/slack/CONFIGURE_SLACK_CLASSIC.md)
+* [Configure MS Teams](docs/teams/CONFIGURE_MS_TEAMS.md)
 
-For starting the bot enter all required variables in [docker-compose.yaml](docker-compose.yaml) and execute:
+For starting the bot we'll use [docker-compose](https://docs.docker.com/compose/install/). 
```

Comment:
Be cautious about this paragraph, we shouldn't recommend docker-compose for production scenarios! It's more a tool for showing how to configure the container and how it works, but the recommendation should be to use a Container Orchestrator, like k8s. 

We should also take a look at high availability scenarios, specially think about: load balancing and fault tolerance.

Please leave a clear point here and create a new issue for addressing the "how-to-deploy" question.